src/Translator.php

```diff
@@ -599,7 +599,7 @@ private function getTransifexSources(string $modulePath): array
         $content = file_get_contents($path);
         $sources = [];
         foreach (preg_split('#\R#u', $content) as $line) {
-            if (preg_match('#source_file\s=\s(?<path>\S+)#', $line, $matches)) {
+            if (preg_match('#source_file\s+=\s+(?<path>\S+)#', $line, $matches)) {
```

Comment:
The colymba/gridfield-bulk-editing-tools tx config has multiple spaces, so it wasn't matching this regex.